siddpubs-conf.bib

```diff
@@ -52,6 +52,13 @@ @string{icml
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{hanjun2019shear,
+  title={Sensing Shear Forces During Food Manipulation: Resolving the Trade-Off Between Range and Sensitivity},
+  author={Hanjun Song, Tapomayukh Bhattacharjee, and Siddhartha S. Srinivasa},
```

Comment:
Please use initials rather than first names (see below for examples).

Reply:
ok, done!